src/chains/strategies/base.strategy.ts

```diff
@@ -35,6 +35,11 @@ export class BaseStrategy extends ChainStrategy {
       case TokenType.CurveLP:
         return getCurveTokenPrice(chain, token.address);
       case TokenType.SushiswapLp:
+      case TokenType.QuickswapLp:
```

Comment:
all univ2 variant tokens are moved to on chain price look ups using the same methodology